### PR TITLE
Add scripts

### PR DESCRIPTION
**Pull request by hokyungh**
*Description of changes:*
Add scripts to run the cli easier to parse.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: chime-sdk-signaling-cpp/demo/cli/scripts/README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ This is helper script to run the cli demo.
 ## Prerequisites
 1. You have installed [python3](https://www.python.org/downloads/) (>=3.6).
 
-2. Install [requests](https://requests.readthedocs.io/en/latest/)
+2. You have install [requests](https://requests.readthedocs.io/en/latest/)
```
Comment (linsang21) [inline chime-sdk-signaling-cpp/demo/cli/scripts/README.md:8]: installed?

**Review by dylonChime — COMMENTED** [review]
Seems `test_cli/scripts/run_cli.py` and `cli/scripts/run_cli.py` are very similar. What is `test_cli/scripts/run_cli.py` for exactly and why can't we combine these scripts?

**Comment by hokyungh** [comment]
> Seems `test_cli/scripts/run_cli.py` and `cli/scripts/run_cli.py` are very similar. What is `test_cli/scripts/run_cli.py` for exactly and why can't we combine these scripts?

One is used for just purely testing - canary - integration tests, so I feel like script can diverge at the end, so separating it out.

File: chime-sdk-signaling-cpp/demo/cli/scripts/README.md (hunk 1 above)
Comment (hokyungh) [reply chime-sdk-signaling-cpp/demo/cli/scripts/README.md:8]: Will fix.

**Comment by hensmi-amazon** [comment]
I'm not convinced that we should have a separate CLI for integration tests; its not like JS SDK where we are using webdrivers and all that stuff which are very sensitive to demo changes. Lets just make that split when we need to.

**Comment by hensmi-amazon** [comment]
I added a more portable bash script in the latest change.

**Pull request closed without merging**